l10n_th_withholding_tax_cert_multi/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Ecosoft Co., Ltd (https://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Thai Localization - Multi Withholding Tax Certificate",
+    "version": "13.0.1.0.0",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/l10n-thailand",
+    "category": "Localization / Accounting",
+    "depends": ["l10n_th_withholding_tax_cert"],
+    "data": ["wizard/create_withholding_tax_cert.xml"],
+    "installable": True,
+    "development_status": "alpha",
```

Comment:
```suggestion
    "development_status": "Alpha",
```
Now, ocabot need capital letter.

Reply:
Edited.